core/spec/lib/spree/core/validators/email_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'spree/core/validators/email'
```

Comment:
is this supposed to stay here?

Reply:
I split it into another commit. Long story short, without the order const loaded, EmailValidator isn't properly loaded and now that the order const isn't loaded when its factory is, this isn't required anymore.